core/src/main/scala-2/zio/temporal/ZSearchAttributeMetaEnumInstances.scala

```diff
@@ -4,6 +4,6 @@ trait ZSearchAttributeMetaEnumInstances {
 
   /** Provides an attribute meta for old [[scala.Enumeration]]
     */
-  def enumeration(enum: Enumeration): ZSearchAttributeMeta.Of[enum.Value, ZSearchAttribute.Keyword, String] =
-    new ZSearchAttributeMeta.KeywordMeta[enum.Value](_.toString, raw => enum.withName(raw))
+  def enumeration(`enum`: Enumeration): ZSearchAttributeMeta.Of[`enum`.Value, ZSearchAttribute.Keyword, String] =
```

Comment:
Was emitting a warning telling that `enum` will be a reserved keyword in Scala3 and should be replaced with a backticked name